ParkingDatabase/src/test/java/com/example/parkingdatabase/ParkingDatabaseApplicationTests.java

```diff
@@ -1,13 +1,67 @@
 package com.example.parkingdatabase;
 
+import com.jayway.jsonpath.internal.function.sequence.Index;
+import org.assertj.core.internal.Arrays;
+import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
 
+import java.util.ArrayList;
+import java.util.List;
+
 @SpringBootTest
-class ParkingDatabaseApplicationTests {
+class ParkingDatabaseApplicationTests {}
+
+    public class AssertjTest {}
 
     @Test
-    void contextLoads() {
+    void test_list_addingcars_works() {
+
+
+        Car car = new Car("ABC23409","Opel","Astra",Color.BLACK);
+        Car car2 = new Car("YFH23409", "Renault", "5W", Color.RED);
+        Car car3 = new Car("OFC23409", "Peugeot", "206", Color.WHITE);
+
+        carRepo.save(car);
+        carRepo.save(car2);
+        carRepo.save(car3);
+
+
+        Iterable<Car> all = carRepo.findAll();
+        all.forEach(System.out::println);
+
     }
 
+    @Test
+    void test_list_removingcars_works() {
+
+        Car car = new Car("ABC23409", "Opel", "Astra", Color.BLACK);
+        carRepo.delete(car);
+
+        Iterable<Car> all = carRepo.findAll();
+        all.forEach(System.out::println);
```

Comment:
Using System.out::println is a nice think for humans to check what happened in test. But in the end you need to write assertions that will check all tests for you automatically